modules/basic/src/main/java/io/oasp/module/basic/common/api/to/OrderByTo.java

```diff
@@ -1,6 +1,4 @@
-package io.oasp.module.jpa.common.api.to;
-
-import io.oasp.module.basic.common.api.to.AbstractTo;
+package io.oasp.module.basic.common.api.to;

```

Comment:
I would recommend to copy the files and keep the original ones as deprecated.
Otherwise you will raise a compatibility issue that will not amuse our users.